samples/host_exerciser/host_exerciser.h

```diff
@@ -324,6 +324,7 @@ class host_exerciser : public test_afu {
         "The Interrupt Vector Number for the device")
         ->transform(CLI::Range(0, 3))->default_val("0");
 
+    app_.add_option("--no-perf", no_perf_, "disable perf counters")->default_val("false");
```

Comment:
can you change to --perf ? instead of --no-perf.

Reply:
fixed